Jenkinsfile.cd

```diff
@@ -202,7 +202,13 @@ def systemTests = { component, releaseVersion ->
                             docker cp node1:/var/lib/indy/sandbox/pool_transactions_genesis ./
                             docker cp ./pool_transactions_genesis indyclient:/home/indy/system/docker_genesis
                             docker cp ./pool_transactions_genesis indyclient:/home/indy/docker_genesis
-                            docker exec -t -u 0 indyclient chown -R indy:indy /home/indy/system
+                            docker exec -t -u 0 indyclient mkdir -p /home/indy/.ssh/
+                            docker cp ./system_tests/system/config indyclient:/home/indy/.ssh/config
+                            docker cp ./system_tests/system/test_key.pub indyclient:/home/indy/.ssh/test_key.pub
+                            docker cp ./system_tests/system/test_key indyclient:/home/indy/.ssh/test_key
+                            docker exec -t -u 0 indyclient chown -R indy:indy /home/indy
```

Comment:
i think it will be enough and cheaper to do chown only for particular dirs in /home/indy (`system` and `.ssh`)